app/actions.ts

```diff
@@ -10,29 +10,6 @@ export const getSearchResultsFromMemory = async (
 ): Promise<BingResults | null> => {
   if (!query || !user?.user) return null;
 
-  console.log("User's query: ", query, user);
-
-  const mem0Response = fetch('https://api.mem0.ai/v1/memories/', {
```

Comment:
why did you remove this? this is imp for storing user informatino.

Reply:
Sorry, I do not see that. Will add again, one sec!